code/game/gamemodes/changeling/powers/armblade.dm

```diff
@@ -62,6 +62,9 @@
 	var/weapType = "weapon"
 	var/weapLocation = "arm"
 
+	var/defend_chance = 30	// The base chance for the weapon to parry.
```

Comment:
If you're going to use this var on multiple items, you may want to put it on the parent class, which in this case would be at least `obj/item/weapon/melee`

Also the `projectile_parry_chance` on everything that has it seems to be a tad high, would probably be better to start it low and increase it if the need presents itself.